packages/apify-extra/src/dataset.ts

```diff
@@ -0,0 +1,339 @@
+import { Actor } from 'apify';
+import { Dictionary, log, sleep } from 'crawlee';
+import bluebird from 'bluebird';
+import type { Dataset } from 'apify-client';
```

Comment:
undeclared dependency (as long as this import is preserved in the d.ts file, we need to declare it in regular deps, not just dev deps)

but maybe better to get rid of this type instead of declaring it, i am sure we have some equivalent in the `@crawlee/types`